fn.go

```diff
@@ -2,13 +2,27 @@ package main
 
 import (
 	"context"
+	"encoding/json"
+	"reflect"
+	"sort"
+
+	"google.golang.org/protobuf/encoding/protojson"
+	"google.golang.org/protobuf/types/known/structpb"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 
 	"github.com/crossplane/crossplane-runtime/pkg/errors"
+	"github.com/crossplane/crossplane-runtime/pkg/fieldpath"
 	"github.com/crossplane/crossplane-runtime/pkg/logging"
+	"github.com/crossplane/function-extra-resources/input/v1beta1"
 	fnv1beta1 "github.com/crossplane/function-sdk-go/proto/v1beta1"
 	"github.com/crossplane/function-sdk-go/request"
+	"github.com/crossplane/function-sdk-go/resource"
 	"github.com/crossplane/function-sdk-go/response"
-	"github.com/crossplane/function-template-go/input/v1beta1"
+)
+
+// Key to retrieve extras at.
+const (
+	FunctionContextKeyExtras = "apiextensions.crossplane.io/extraResources"
```

Comment:
Really don't care too much either way about the key or name of the function. Just change it if ya like.